nix/modules/devShells.nix

```diff
@@ -173,6 +173,7 @@
             export PATH="$CARGO_INSTALL_ROOT/bin:$PATH"
             export NIX_PATH="nixpkgs=${pkgs.path}"
             export PS1='\n\[\033[1;34m\][rustDev:\w]\$\[\033[0m\] '
+            export LIBCLANG_PATH="${pkgs.llvmPackages.libclang.lib}/lib"
```

Comment:
This works on MacOS and Linux?

Is this something we should consider in the new Holonix or are you not having to make these tweaks there?

Reply:
Confirmed it works on macos. Looks like we have some unrelated issues building holochain on the new holonix -- rust 1.80 now throws errors for unrecognized cfg flags. Should we leave that for another PR?

Reply:
Yes, we're still running on Rust 1.78 in the repo.

Reply:
I've fixed a bunch of that separately, once we have both PRs in we can try again in the new Holonix